### ISSUE TITLE
Bubble timeline and cat ears

### ISSUE THREAD

**Issue by flufferyomada** [TheEssem/mastodon#4]
### Pitch

misskey style cat ears. 
![image](https://github.com/TheEssem/mastodon/assets/74445505/8a5f05f3-6e3b-4513-907c-9f94900a4fa2)

pleroma/akkoma (not sure who did it) local bubble showcase 
![image](https://github.com/TheEssem/mastodon/assets/74445505/db999543-5651-4910-933c-7a714f439f3d)

### Motivation

cats ear: because. it will be very useful to the users of chuckya (source is that i had it come to me as a realization in a delusion)
local bubble: i think it would be cool for it to exist, just do that cat ears bro

**Comment by Sqaaakoi** [comment]
i agree we need these, the cat ears are just fun and silly, and the local bubbles are more important IMO as they would expand communities (sometimes doomscrolling the local feed gets just a little bit too lonely)

**Comment by TheEssem** [comment]
Actively looking into adding the bubble timeline, though it seems to be a bit more difficult than anticipated due to the way that Mastodon handles timelines.

As for the cat ears, I'm still undecided...

**Comment by ch0ccyra1n** [comment]
I've been thinking about this, and perhaps one enhancement you could make for the bubble timeline is to allow admins to add groups to the Bubble Timeline. For example, my instance focuses on stuff in my local area and I would want to add the Lemmy community for my local instance to the bubble timeline.

**Comment by flufferyomada** [comment]
> I've been thinking about this, and perhaps one enhancement you could make for the bubble timeline is to allow admins to add groups to the Bubble Timeline. For example, my instance focuses on stuff in my local area and I would want to add the Lemmy community for my city to the bubble timeline.

more integrated lemmy connections should be considered, maybe like a friendica approach, I just barely know ruby and I can't do it myself. I have alot of ideas and alot of parity from other fedi software that would be nice and a pain to have in one but don't really have the skills to do it.

**Comment by TheEssem** [comment]
The bubble timeline has now been implemented.

**Comment by ch0ccyra1n** [comment]
Hmm, I seem to be having an issue where the Bubble Timeline ends up being the exact same as the Other Servers timeline.

**Comment by ch0ccyra1n** [comment]
### Top two posts from other servers timeline
![image](https://github.com/user-attachments/assets/cbe269c2-2bab-4e0b-a87a-94381fbc8433)

### Top two posts from bubble servers timeline
![image](https://github.com/user-attachments/assets/3c6833f3-cfed-43fa-bef4-263c53e99de2)

For reference, my list of servers in the bubble list is literally empty, so surely it should show absolutely zero posts in the bubble timeline, right?

**Comment by TheEssem** [comment]
@ch0ccyra1n This is a known bug; unfortunately though, I've been unable to reproduce the issue. Any data (e.g. request logs) that can be useful to fix this would be greatly appreciated, as it seems like a lot of people are having this issue.
